### DISCUSSION TITLE
Getting incorrect system IDs via SDRTrunk

### DISCUSSION THREAD

**Discussion by SimonSheehan** [chuot/rdio-scanner#347] [Q&A] [answered]
Hello,

Having issues with using dirwatch to pull from SDRTrunk right now. I have one system setup that works flawlessly. However, the other two are showing up with `no matching system/talkgroup`, despite the systemID matching in Rdio's settings and I can see that same system ID on SDRTrunk when checking the control channel. I've confirmed I have the right talkgroups added too. 

Wondering where I'm going wrong here and if anyone may know what to look for? Thank you for your time. Error below for reference:

```
2023/04/22 21:11:50 newcall: system=2 talkgroup=700 file=longaudiofilename.mp3 no matching system/talkgroup

2023/04/22 21:13:15 newcall: system=1 talkgroup=11002 file=longaudiofilename2.mp3 no matching system/talkgroup
```

**Comment by Clark985** [comment, marked as answer]
Are you using the API key/downstream setup to link multiple systems together?

On a given SDRtrunk + RDIO instance, the system name in SDRtrunk's channel config is matched to the system Label in RDIO-Scanner.
The system ID number in RDIO is just an arbitrary number that you assign that system.  BUT, if you're linking multiple together, that has to jive between RDIO instances.

I'm running a main instance with 7 remote receive nodes all linked together.  I had to have the people running a couple of the receive nodes change their system IDs to match mine to make it all work together (not using the default of "1").
